spring-cloud-gateway-server/src/main/java/org/springframework/cloud/gateway/filter/factory/cache/LocalResponseCacheFilter.java

```diff
@@ -36,11 +36,11 @@
  * @author Ignacio Lozano
  * @author Marta Medio
  */
-public class GlobalLocalResponseCacheGatewayFilter implements GlobalFilter, Ordered {
+public class LocalResponseCacheFilter implements GlobalFilter, Ordered {
```

Comment:
Unfortunately, we can't change the name of this class now because it has already been released.

Reply:
I left the previous name: https://github.com/spring-cloud/spring-cloud-gateway/pull/2895/commits/ee51aed27690e8c98e39cdf06d33e81f769ed655